pages/features/autoresponder-email.vue

```diff
@@ -122,10 +105,12 @@ import FeatureDetail from '../../components/FeatureDetail.vue'
 import Testimonial from '@/components/Testimonial.vue'
 import { allTestimonials } from '@/constants/testimonials'
 import TemplateSection from '../../components/TemplateSection.vue'
+import UseCaseFeatureShowcase from "../../components/use-case/FeatureShowcase.vue"
```

Comment:
try to always use absolute paths

Reply:
I know auto import in vs-code adds relative path by itself 😅

Reply:
Okay, I will change these imports.